samples/app-hello-world/nodejs/README.md

```diff
@@ -20,11 +20,17 @@ urlFragment: officedev-microsoft-teams-samples-app-hello-world-nodejs
 * Tabs
 * Bots
 * Messaging Extensions
+* Element RelationShip Property
```

Comment:
I'm new to sample app creation, but why do we not create a separate RT sample app yet use existing hello-world app to illustrate RT usage? This might create confusion to developers.

Reply:
Also so far only elementRelationshipSet is covered. I don't see "hostMustSupportFunctionalities" yet.

Reply:
Fixed